versions/1/manifest.spec.yml

```diff
@@ -47,8 +47,10 @@ spec:
       type: string
       enum:
       - integration
+      - rules
```

Comment:
I'm hesitant to introduce a new package type this way (silently), also `rules` sounds generic to me. So far the integration type is the only supported one and I'm not sure what would be implications of introducing this.

Any thoughts, @ruflin?

Reply:
I think what we need to discuss here is how these packages should show up in the package Manager.

@rw-access I would take this to a separate PR.

Reply:
For this package type, EPR is essentially used for storage but not much else.

Here's some ways where they differ:
- Detection rules packages aren't installed in the traditional manner via the Fleet pages,
- The rules in the package are always "installed" to a specific space
- Detection rules packages shouldn't show up in the Fleet Integrations page and they should be hidden. `internal: true` might be a better way to accomplish this than a new type

Reply:
Since these packages have no relationship with Agent and would be hidden from the Integrations page, I didn't think `type: integration` was appropriate. Something like `type: artifact` seems more appropriate and still generic

Reply:
This is helpful. Do you expect that rules can also be added for example to an `nginx` package for nginx specific rules in the future? What happens if someone adds rules to a package?

Reply:
We had a lot of discussions, and the `rules` custom type for a package isn't necessary. Thanks for helping me get there, @ruflin.
Updated the PR so this thread is no longer relevant